spec/models/reports/drug_stock_calculation_spec.rb

```diff
@@ -223,4 +227,21 @@
       expect(instance.patient_days_coefficients("state_that_does_not_exist")).to eq nil
     end
   end
+
+  describe "#consumption" do
+    it "uses the redistribution numbers for consumption calculation" do
```

Comment:
This might be too specific to the current PR. How about something like `calculates consumption using stock in hand, received, and redistributed`?